mopidy/core/actor.py

```diff
@@ -136,6 +140,103 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def setup(self):
```

Comment:
So am I right that the ability to save/restore state is currently not going to be available via JSON-RPC? I've got a user case for it in Musicbox, is there a particular reason we don't want to expose it?